test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -158,6 +158,9 @@ func TestStandaloneWorkspacesVirtualWorkspaces(t *testing.T) {
 	if len(framework.TestConfig.KCPKubeconfig()) != 0 {
 		t.Skip("Skip testing standalone when running against persistent fixture to minimize test execution cost for development")
 	}
+
+	t.Skip("TODO reenable this test as part of the work to resolve https://github.com/kcp-dev/kcp/issues/2056")
```

Comment:
💣 🔴  to be checked: the redirect we had, does the clear out the bearer token from the incoming loopback client? If not, that's a serious security issue. The bearer token is system:masters against the shard.

Reply:
Won't be an issue going forward. The redirect will be entirely replaced by a proxy that uses a proper client cert (not loopback).